plugins/inputs/activemq/README.md

```diff
@@ -5,6 +5,13 @@ API.
 
 ## Configuration
 
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md
```

Comment:
Maybe link to `CONFIGURATION.md#input-plugins` or `CONFIGURATION.md#plugins` to be more specific